src/puppetlabs/puppetdb/command.clj

```diff
@@ -69,7 +69,7 @@
             [puppetlabs.puppetdb.utils :as utils]
             [slingshot.slingshot :refer [try+ throw+]]
             [puppetlabs.puppetdb.mq-listener :as mql]
-            [puppetlabs.puppetdb.command.constants :refer [command-names]]
+            [puppetlabs.puppetdb.command.constants :as cmd-consts]
```

Comment:
I suspect this was mine originally -- for what it's worth, both here, and with respect to the q/foo changes, I sometimes prefer the shorter names because I find the prefixes "noisy" unless they're part of the meaning (or disambiguating) (say for stock/open stock/reduce).  For me, something like `cmdref->entry` is clear enough, and past a point, prefixes can make the code somewhat harder to read.